src/app/birthday/birthday.service.spec.ts

```diff
@@ -10,7 +10,38 @@ describe('Service: Birthday', () => {
     });
   });
 
-  it('should ...', inject([BirthdayService], (service: BirthdayService) => {
+  it('Service should exist', inject([BirthdayService], (service: BirthdayService) => {
     expect(service).toBeTruthy();
   }));
+
+  describe('getBirthdayFor behavior', () => {
+    it('Should return Date object', fakeAsync(() => {
+      let methodReturnedValue: Date;
+  
+      getService()
+        .getBirthdayFor('Dima')
+        .subscribe(event =>  methodReturnedValue = event);
```

Comment:
Instead of manually storing value from a callback you can use spies (`jest.fn()`)
And then you can just pass it as a callback and expect it to be called with some value.